### ISSUE TITLE
Question about disabling Plex 

### ISSUE THREAD

**Issue by jrcalvert** [Cloudbox/autoscan#205]
In the documentation it states that "it is advised to turn off Plex's Update my library automatically feature", which obviously makes sense, but I would like some clarification over what settings should be set in plex, i.e.

- Scan my library automatically - I have disabled (unticked) this
- Run a partial scan when changes are detected - Should I also disable this?
- Scan my library periodically (e.g. daily) - Should I leave this enabled?

Thanks

**Comment by mpk99** [comment]
pretty sure you can disable all three.  I have it setup that way and it's only scanning the dirs that i setup with sonarr/radarr and inotify and it's working perfect.

**Comment by ninidas** [comment]
Does aAutoscan do a partial scan ? Like scanning only the change and not the entire library ?

**Comment by ninidas** [comment]
> pretty sure you can disable all three. I have it setup that way and it's only scanning the dirs that i setup with sonarr/radarr and inotify and it's working perfect.

Does Autoscan do a partial scan ? Like scanning only the change and not the entire library ?

**Comment by m-rots** [comment]
> > pretty sure you can disable all three. I have it setup that way and it's only scanning the dirs that i setup with sonarr/radarr and inotify and it's working perfect.
> 
> Does Autoscan do a partial scan ? Like scanning only the change and not the entire library ?

Indeed, Autoscan only scans the file that has changed, not the entire library.
